test/common/http/http2/http2_frame.h

```diff
@@ -99,6 +99,9 @@ class Http2Frame {
     std::string value_;
   };
 
+  // Make client request stream ID out of the given ID in the host byte order
+  static uint32_t makeClientStreamId(uint32_t stream_id) { return (stream_id << 1) | 1; }
+
```

Comment:
Perhaps a quick comment about client ids needing to be odd. Implied from the bitshift/or, but maybe add reasoning.

Reply:
Could even link rfc specification like you did in pr description.

Reply:
+1 I'd even put "Use this function to create client stream ids for methods creating HTTP/2 frames." since this is a usage change from the before "pseudo-index"

Reply:
Done.